_data/vendors.yml

```diff
@@ -117,11 +117,11 @@
 
 - name: GitHub
   url: https://www.github.com
-  base_pricing: $9 per u/m
+  base_pricing: $4 per u/m
   sso_pricing: $21 per u/m
-  percent_increase: 133%
+  percent_increase: 525%
```

Comment:
```suggestion
  percent_increase: 425%
```

Reply:
Ah is it meant to be the percentage of the base that the difference represents? Makes sense then.